Use grayscaled icon for "Reset another branch to here"

zvirja opened the pull request:
## Proposed changes

I often use `Reset another branch to here` action as a way to sync an inactive branch. Currently in a context menu we have two very similar options - `Reset current branch to here` and `Reset another branch to here` and both of them use the same color. I found that this way I cannot use icon as a way to quickly identify the menu entry, but have to resort to text reading and remembering the position, which is slow and slightly annoying.

By using a different icon for `Reset another branch to here` now I can quickly pick the action from list

I used a grayscaled version of the existing icon as we reset inactive branch. So grey should nicely represent the "inactive" idea.

## Screenshots <!-- Remove this section if PR does not change UI -->

### Before

![image](https://user-images.githubusercontent.com/1074182/169010958-f268cc34-5cfa-4c91-8e1d-76a45e320a06.png)

### After

![image](https://user-images.githubusercontent.com/1074182/169010041-eb7e152d-15bd-4cee-a284-5023e951cc15.png)

## Test methodology

Manual testing

## Merge strategy

<!-- Change the following if the merge strategy should be changed:
- Squash merge (maintainer to decide merge message, PR submitter should cleanup commits/messages at PR approval).
- Rebase merge (PR submitter must change the commit message for the last commit).
- Merge commit. (PR submitter to rebase and squash before merges).
- To be decided later.
The maintainer may still request the contributor to squash and rebase, to make sure that merges and commit messages are clarified.
-->

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

RussKie reviewed (approved):
I too periodically pick the wrong reset option ;))

gerhardol commented:
Please sign the cla
The .png had incorrect attributes and would not scale correctly, similar to #7942
I had to use the tool to check and pushed an update.
I would rather keep the options next to each other.

gerhardol reviewed (changes requested):
The icon does not scale correctly

RussKie commented on UnitTests/GitUI.Tests/ResourcesTests.cs:
```diff
@@ -0,0 +1,77 @@
+        public void PngResources_ShouldNotContainDPIInformation()
+        {
+            // arrange
+            ResourceSet resourceSet = Images.ResourceManager.GetResourceSet(CultureInfo.InvariantCulture, true, false);
```
`ResourceSet` is [disposable](https://docs.microsoft.com/en-us/dotnet/api/system.resources.resourceset?view=net-6.0)
```suggestion
            using ResourceSet resourceSet = Images.ResourceManager.GetResourceSet(CultureInfo.InvariantCulture, true, false);
```

Also please add argument names for `true` and `false`.

zvirja replied:
Actually it should not be disposed, as otherwise a lot of tests are failing. Apparently, you retrieve a global resource, but not own it 😕 
Dropped a comment on that

zvirja commented:
@gerhardol Fixed:
- Signed CLA
- Fixed year for previous CLA entry
- Fixed new PNG resource
- Added unit test for all PNG resources in `Images` resource set and also fixed a couple of existing violations.

Please review again 😊

RussKie commented on UnitTests/GitUI.Tests/ResourcesTests.cs:
```diff
@@ -0,0 +1,77 @@
+                // Skip header
+                cursor = cursor[header.Length..];
+
+                var sectionTypes = new List<string>();
```
[nit]
```suggestion
                List<string> sectionTypes = new();
```

RussKie commented on UnitTests/GitUI.Tests/ResourcesTests.cs:
```diff
@@ -0,0 +1,77 @@
+    public class ResourcesTests
+    {
+        [Test]
+        public void PngResources_ShouldNotContainDPIInformation()
```
🚀

zvirja commented:
@gerhardol I've added another unit test to verify the function itself. It's a 1 pixel base64 encoded PNG and grabbed correct list using another tool. Should be enough to be confident 😉

gerhardol reviewed (approved):
Thanks!

RussKie commented:
Thank you
